distribution/ecs/quickwit/service/variables.tf

```
@@ -41,6 +41,7 @@ variable "service_config" {
     desired_count = optional(number, 1)
     memory        = number
     cpu           = number
+    storage       = optional(number, 21)
```

```suggestion
    ephemeral_storage_size_gib       = optional(number, 21)
```

I had a use case yesterday for also attaching some form of persistent storage (ebs or efs), so I think "ephemeral" is less ambiguous.